prototypes/signal.lua

```diff
@@ -0,0 +1,125 @@
+local capture_tint = {0.9,0.9,0.9}
+local target_tint = {0.1,0.6,0.2}
+
+data:extend({
+    {
+        type = "item-subgroup",
+        name = "virtual-signal-TLBE",
+        group = "signals",
+        order = "z",
+    },
+    {
+        type = "virtual-signal",
+        name = "signal-capture-north-west",
+        icons = {
+            {
+                icon = "__TLBE__/graphics/box-north-west.png",
+                tint = capture_tint
+            }
+        },
+        icon_size = 32,
+        icon_mipmaps = 1,
+        subgroup = "virtual-signal-TLBE",
+        order = "a"
```

Comment:
As this is a virtual-signal it will never show up in the UI? In this case I suppose we can drop `order` as it is not doing anything

Reply:
They seem to show up in the signals tab - I couldn't see how to hide them entirely

Reply:
I though I checked and didn''t saw them. I guess I didn't looked hard enough. But I found them now :smile: 

I found a [`selectable_in_game`](https://wiki.factorio.com/Prototype/Entity#selectable_in_game) property, maybe we can use that to get rid of them (I am just guessing here, as I never used signals and the documentation is kind of lacking here)

Reply:
I'll test this

Reply:
Nope. No dice. So I've fixed their ordering so that the two colors of corners don't interleave. I'll also add text for them to get rid of the nasty keys.

Reply:
that's too bad... Now there are some weird signals in the GUI. But I guess it won't matter too much